themebuilder/data/migration-metadata/migration-metadata.js

```diff
@@ -96,39 +96,21 @@ module.exports = {
         '@radiogroup-checked-bg',
         '@radiogroup-border-color',
         '@radiogroup-bg',
-        '@radiobutton-active-bg',
```

Comment:
Why we have differences with v18.2 here?

Reply:
https://github.com/DevExpress/DevExtreme/pull/7611/files this?

Reply:
Oh I missed it.